### PR TITLE
Leddie24/component cleanup

### PR DESCRIPTION
**Pull request by leddie24**
List of changes

- Changed commandbar text (Bug 239147: CommandBar text doesn't make sense - we should remove the sentence about the NavBar.)
- Stripped out html comments in example code (Bug 239148:Copyright text is being pulled in for each template in handlebars).
- Changed title of localhost docs (Bug 239150:Change name of project in locally built docs site to Office UI Fabric JS)

### CODE REVIEW

**Comment by mikewheaton** [comment]
Approved 🍨

**Comment by ericthompson** [comment]
Approved :ear:

File: src/documentation/templates/componentDemo.html [hunk 1]
```diff
@@ -20,4 +20,18 @@
...
+      var commentRegex = /(<!--.+-->)+/g;
+      var matches = [];
+
+      for (var i = 0; i < codeBlocks.length; i++) {
+        var code = codeBlocks[i].innerText;
+        codeBlocks[i].innerText = code.replace(commentRegex, '');
+      }
+    }
+
+    stripComments();
```
Comment (battletoilet) [inline src/documentation/templates/componentDemo.html]: I feel like this process should happen on the client side but instead with a Handlebars plugin or something similar.

Comment (leddie24) [inline src/documentation/templates/componentDemo.html]: @battletoilet ready for review again, thanks!

**Pull request merged**